### PR TITLE
Acept challege 

### PR DESCRIPTION
**Pull request by lolivera-eb**
The challenged player accepts the challenge and the web is notified that the game started
**Create**
def accept_challenge 
test accept_challenge
Ticket : thttps://eventbrite.atlassian.net/browse/EDA-1243

### CODE REVIEW

File: server/server.py [hunk 1]
```diff
@@ -50,6 +51,14 @@ def notify_game_created(challenge_id, game_id):
     )
 
 
+def accept_challenge(data):
+    game_id = data['data']['game_id']
```
Comment (julianandres-eb) [inline server/server.py]: Are you preventing a `KeyError` here?

File: server/server.py [hunk 2]
```diff
@@ -60,6 +69,8 @@ async def session(websocket: WebSocket, token):
     try:
         while True:
             msg = await websocket.receive_text()
+            if msg['action'] == 'accept_challenge':
+                accept_challenge(msg)
```
Comment (julianandres-eb) [inline server/server.py]: If you're going to add more action, it's better to have them parameterized or some sort of `Factory`

File: server/server.py (hunk 1 above)
Comment (lolivera-eb) [reply server/server.py]: fixing up

File: server/server.py (hunk 2 above)
Comment (lolivera-eb) [reply server/server.py]: it is something that we are going to solve.

File: server/exception_code.py [hunk 3]
```diff
@@ -0,0 +1 @@
+GAMEIDERROR = 1
```
Comment (julianandres-eb) [inline server/exception_code.py]: `GAMEID_ERROR = 'GAMEID_ERROR'` will be better. A number says nothing about the error

File: server/server.py [hunk 4]
```diff
@@ -20,6 +23,19 @@ def notify_game_created(challenge_id, game_id):
     )
 
 
+async def accept_challenge(data, client):
+    game_id = data.get('data', {}).get('game_id')
+    if game_id is None:
+        await notify_error_to_client(
+            client,
+            GameIdException,
```
Comment (julianandres-eb) [inline server/server.py]: I think you should do `str(GameIdException)`

File: server/exception_code.py (hunk 3 above)
Comment (lolivera-eb) [reply server/exception_code.py]: done

File: server/server.py (hunk 4 above)
Comment (lolivera-eb) [reply server/server.py]: done !

**Review by julianandres-eb — APPROVED** [review]

**Pull request merged**